documentation/modules/creating-and-running-a-migration-quick-plan.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="creating-running-migration0-quick-plan_{context}"]
+= Creating and running a migration quick plan
+
+You can create a migration _quick plan_ after you have created least at one of each of the components of a migration plan. After you select the virtual machines (VMs) you want to migrate, {project-first} suggests other plan components, such as network and storage mapping, based on the components you have created. Although you are always free to reject a suggested quick plan, this feature can save you time and effort.
+
+.Procedure
+
+. In the {ocp} web console, click *Providers for virtualization* and then click the *VMs* column of the provider whose virtual machhines (VMs) you want to migrate.
```

Comment:
There are at least 3 ways to get to the Providers -> VMs tab. But I guess this one is the most "classic".

Reply:
@rszwajko Yes -- you might have noticed that we try to document one way to perform a task, especially when it is a task such as accessing something in the UI. If yoiu think we skipped something important, please let us know.